filebeat/docs/inputs/input-docker.asciidoc

```diff
@@ -49,6 +49,11 @@ is `/var/lib/docker/containers`.
 Reads from the specified streams only: `all`, `stdout` or `stderr`. The default
 is `all`.
 
+===== `containers.partial`
+
+Enable partial messages joining. Docker splits log lines larger than 16k bytes, enable this
+to join them while reading (default: false).
```

Comment:
I think it would be useful to specify that it works by joining lines that don't end with \n.

Reply:
uhm that's something specific to the `json-file` format, it's not the user who adds the `\n`. Do you think it's worth it? If yes, I would explain the whole story about how the format works